tools/pd-backup/pdbackup/backup.go

```diff
@@ -73,7 +73,7 @@ func GetBackupInfo(client *clientv3.Client, pdAddr string) (*BackupInfo, error)
 
 	backInfo.AllocIDMax = allocIDMax
 
-	timestampPath := path.Join(rootPath, "timestamp")
+	timestampPath := path.Join(rootPath, "allocator", "global", "timestamp")
```

Comment:
I think we should not change the global timestamp's store path in case of backward compatibility.